src/AutosizeInput.js

```diff
@@ -36,7 +36,7 @@ const copyStyles = (styles, node) => {
 	node.style.textTransform = styles.textTransform;
 };
 
-const isIE = (typeof window === 'undefined' && window.document) ? false : /MSIE |Trident\/|Edge\//.test(window.navigator.userAgent);
+const isIE = typeof window === 'undefined' ? false : /MSIE |Trident\/|Edge\//.test(window.navigator.userAgent);
```

Comment:
`var isIE = typeof window !== 'undefined' && (window.document ? false : /MSIE |Trident\/|Edge\//.test(window.navigator.userAgent));` would be fine too